docs/pages/lighting-setups.rst

```diff
@@ -0,0 +1,197 @@
+    :end-before: # [/scene]
+
+
+
+.. |im1| image:: images/lighting-setups-images/1.jpg
+   :width: 100%
+
+.. |im2| image:: images/lighting-setups-images/2.jpg
+   :width: 100%
+
+.. |cp1| replace:: After loading with DEFAULT_LIGHTING_KEY.
+
+.. |cp2| replace:: After swapping to custom "my_scene_lighting".
+
++---------+-----------+
+|  |im1|  +   |im2|   +
+|  |cp1|  +   |cp2|   +
++---------+-----------+
+
+
+`Object Lighting`_
+==================
+
+By default, object assets are loaded with Phong shading compatability. This is not ideal for assets with illumination baked into textures.
+Objects can be loaded for Flat shading by using the asset template's **PhysicsObjectAttributes.setRequiresLighting** function *before* loading the asset.
+Alternatively, this option can be set in the object template's configuration file:
+
+E.g. in *my_object.phys_properties.json*
+
+.. code:: json
+
+    {
+        "render mesh": "my_object.glb",
+        "requires lighting": false
+    }
+
+By default, new objects with Phong shading enabled are added to the scene with the **habitat_sim.gfx.DEFAULT_LIGHTING_KEY** LightSetup.
+
+.. include:: ../../examples/tutorials/lighting_tutorial.py
+    :code: py
+    :start-after: # [example 2]
+    :end-before: # [/example 2]
+
+.. image:: images/lighting-setups-images/3.jpg
+    :width: 45%
+
+We can modify the default LightSetup by setting with an empty key.
+
+.. include:: ../../examples/tutorials/lighting_tutorial.py
+    :code: py
+    :start-after: # [example 3]
+    :end-before: # [/example 3]
+
+.. image:: images/lighting-setups-images/4.jpg
+    :width: 45%
+
+Newly added objects will use the current default lighting.
+
+.. include:: ../../examples/tutorials/lighting_tutorial.py
+    :code: py
+    :start-after: # [example 4]
+    :end-before: # [/example 4]
+
+.. image:: images/lighting-setups-images/5.jpg
+    :width: 45%
+
+`Multiple Light Setups`_
+========================
+
+To use multiple custom lighting setups at the same time, simply give them a name on creation.
+
+.. include:: ../../examples/tutorials/lighting_tutorial.py
+    :code: py
+    :start-after: # [example 5]
+    :end-before: # [/example 5]
+
+To use this a light setup, pass in the name as a parameter to **Simulator.add_object**.
+
+.. include:: ../../examples/tutorials/lighting_tutorial.py
+    :code: py
+    :start-after: # [example 6]
+    :end-before: # [/example 6]
+
+.. image:: images/lighting-setups-images/6.jpg
+    :width: 45%
+
+You can get a copy of an existing configuration with **Simulator.get_light_setup**.
+
+.. include:: ../../examples/tutorials/lighting_tutorial.py
+    :code: py
+    :start-after: # [example 7]
+    :end-before: # [/example 7]
+
+Updates to existing light setups will update all objects using that setup
+
+.. include:: ../../examples/tutorials/lighting_tutorial.py
+    :code: py
+    :start-after: # [example 8]
+    :end-before: # [/example 8]
+
+.. image:: images/lighting-setups-images/7.jpg
```

Comment:
Why do all the images have an explicit white border? Padding can be done much more flexibly on the markup side. (I wanted to experiment with layouts a bit, but the white border made it all ugly :sweat_smile:)

The style also applies slight rounding to all image corners, but the white border makes that invisible.

Reply:
This is a result of pyplot saving images with default padding. Should be better now.